lexers/l/logos.go

```diff
@@ -0,0 +1,29 @@
+package l
+
+import (
+	"regexp"
+
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+var logosAnalyserKeywordsRe = regexp.MustCompile(`%(?:hook|ctor|init|c\()`)
+
+// Logos lexer.
+var Logos = internal.Register(MustNewLexer(
+	&Config{
+		Name:      "Logos",
+		Aliases:   []string{"logos"},
+		Filenames: []string{"*.x", "*.xi", "*.xm", "*.xmi"},
+		MimeTypes: []string{"text/x-logos"},
+	},
```

Comment:
This one has `priority` defined. I do not know, if we kept an eye on it, but it would be crucial to port this to chroma as well. It is actively used in wakatime-cli. I also added an issue to verify this for any other lexer, where we might have forgotten: https://github.com/wakatime/wakatime-cli/issues/230

Reply:
Good question. As it's a simple new line to be added when needed let's also port this property to other lexers as well.